app/webpacker/components/Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt/index.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Container, Grid, Header, Message, Step,
+} from 'semantic-ui-react';
+import { QueryClient, useQuery } from '@tanstack/react-query';
+import getDetailsBeforeAnonymization from './api/getDetailsBeforeAnonymization';
+import Loading from '../../../Requests/Loading';
+import Errored from '../../../Requests/Errored';
+import VerifyAnonymizeDetails from './VerifyAnonymizeDetails';
+import AnonymizeAction from './AnonymizeAction';
+import ActionItems from './ActionItems';
+import NonActionItems from './NonActionItems';
+
+const ANONYMIZATION_QUERY_CLIENT = new QueryClient();
+
+const STEPS = [
+  '1. Verify details to be anonymized',
+  '2. Non-action items to review',
+  '3. Action items to perform',
+  '4. Anonymize Action',
+];
```

Comment:
Are the numeral prefixes necessary? SemUI has `ordered` for the `Step` component: https://react.semantic-ui.com/elements/step/#groups-ordered, which should do the trick